src/java/org/apache/cassandra/service/accord/AccordResult.java

```diff
@@ -142,6 +143,8 @@ else if (isTxnRequest && coordinationFailed instanceof TopologyMismatch)
             }
             else
             {
+                logger.error("Unexpected exception", fail);
```

Comment:
I was seeing that logs were clean but exceptions were happening, and found that this code block was hiding all these exceptions.  The error isn't a `CoordinationFailed` so its not expected, so we should be loud!  I found several unexpected exceptions by adding this in